overrides/macos-override.json

```diff
@@ -43,7 +43,8 @@
             "state": "enabled"
         },
         "trackingParameters": {
-            "state": "disabled"
+            "state": "enabled",
+            "minSupportedVersion": "0.22.3"
```

Comment:
Was going to say I thought we still needed to strip this manually but the other code in index.js does this... thanks!